src/home_robot/home_robot/perception/constants.py

```diff
@@ -281,6 +297,99 @@ def num_sem_categories(self):
         return 16
 
 
+languagenav_2categories_indexes = {
+    1: "target",
+    2: "landmark",
+}
+
+languagenav_2categories_padded = (
+    ["."] + [languagenav_2categories_indexes[i] for i in range(1, 3)] + ["other"]
+)
+
+languagenav_2categories_legend_path = str(
+    Path(__file__).resolve().parent / "rearrange_3categories_legend.png"
+)
+
+languagenav_2categories_color_palette = [255, 255, 255] + list(
+    d3_40_colors_rgb[1:3].flatten()
+)
+languagenav_2categories_frame_color_palette = languagenav_2categories_color_palette + [
+    255,
+    255,
+    255,
+]
+
+languagenav_2categories_map_color_palette = [
```

Comment:
It might be good to document this a little more - it's sort of unclear how to edit this